### DISCUSSION TITLE
Horizontal scrollbar in the search results

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#1999] [General]
A horizontal scrollbar appears in the search results output (the results table is wider than the space to display it).
Is there a way to fix the css to display it correctly (without the scroll bar)?

![2023-02-10_12-06-06](https://user-images.githubusercontent.com/16667173/218050180-cdf569f2-f66d-4e04-a610-6d4c9755ab76.png)

**Comment by AndreasK79** [comment]
What do you mean by display it correctly? If the scrollbar is not there, you can't access the button to the right.

**Reply by R1BLH** [reply]
I mean display whole table without scrollbar. To make wider placeholder (class?) for table.

**Reply by AndreasK79** [reply]
Not sure how I would fix that. I guess that would be possible by making everything smaller. What desktop resolution are you using?

**Reply by AndreasK79** [reply]
I have made wider custom themes available, but not sure if those would help you.

**Reply by R1BLH** [reply]
> Not sure how I would fix that. I guess that would be possible by making everything smaller. What desktop resolution are you using?

1920x1080 (this is typical for my desktops).

**Reply by AndreasK79** [reply]
Right, then these might work for you: https://github.com/AndreasK79/cloudlog_wide_custom_themes

**Reply by R1BLH** [reply]
> I have made wider custom themes available, but not sure if those would help you.

The same situation is with the main page. Placeholder for the log table is a bit smaller than table width.
![2023-02-10_12-52-46](https://user-images.githubusercontent.com/16667173/218060858-147827a5-f695-4911-a17e-9e0120f61741.png)

**Reply by R1BLH** [reply]
> Right, then these might work for you: https://github.com/AndreasK79/cloudlog_wide_custom_themes

Great! Thank you!

**Reply by R1BLH** [reply]
> Right, then these might work for you: https://github.com/AndreasK79/cloudlog_wide_custom_themes

One more question.
Could you advise what I should change to make the placeholder for the QSL image smaller (to remove the vertical scroll bars)?

![2023-02-15_14-39-47](https://user-images.githubusercontent.com/16667173/219018363-82849bd2-6f8c-4ad9-8372-fb5d910e029f.png)

**Reply by R1BLH** [reply]
@AndreasK79 
Now image of the QSL has 100% width. It would be better with 55% width, see the screenshots.

100% (now)
![2023-02-16_10-34-33](https://user-images.githubusercontent.com/16667173/219298637-e2844356-add4-4dea-969a-897ac31b4f98.png)

55% 
![2023-02-16_10-35-36](https://user-images.githubusercontent.com/16667173/219298633-78e7f0fa-fe6c-4380-9bd4-72a9cbf3ee92.png)

**Reply by AndreasK79** [reply]
It should really be responsive, so that it adjusts to the size of the popup. I have no solution to fix this.

**Reply by R1BLH** [reply]
> It should really be responsive, so that it adjusts to the size of the popup. I have no solution to fix this.

I added another display class for QSL (in /assets/css/darkly_wide/bootstrap.min.css) similar to "w-100" -- "w-qsl". 
`w-qsl{width:55%!important}`
And replaced use of class "w-100" in QSL display (in /application/views/view-log/qso.php, line 517):
`echo '<img class="d-block w-qsl" src="' . base_url() . '/assets/qslcard/' . $image->filename ." alt="QSL picture #'. $i++.'">';`

The result is in the screenshots.

![2023-02-17_10-57-09](https://user-images.githubusercontent.com/16667173/219590062-07e6d165-4b3c-487d-a2d4-3016b796be28.png)
![2023-02-17_10-57-48](https://user-images.githubusercontent.com/16667173/219590069-f0c767e9-3473-40cb-86b0-182c87fdf5f9.png)
![2023-02-17_10-58-43](https://user-images.githubusercontent.com/16667173/219590073-68c7e76f-cde0-4572-bdb4-4dd2350d9fd9.png)

**Comment by magicbug** [comment]
This isn't a bug, the titles for the columns are to long hence the scrollbar, there's nothing to fix, but you could try a different theme
